sentry-actix/src/lib.rs

```diff
@@ -411,4 +403,41 @@ mod tests {
             assert_eq!(request.method, Some("GET".into()));
         }
     }
+
+    /// Ensures transaction name can be overridden in handler scope.
+    #[actix_rt::test]
+    async fn test_override_transaction_name() {
+        let events = sentry::test::with_captured_events(|| {
+            block_on(async {
+                #[get("/test")]
+                async fn original_transaction(_req: HttpRequest) -> Result<String, Error> {
+                    // Override transaction name
+                    Hub::current()
+                        .configure_scope(|scope| scope.set_transaction(Some("new_transaction")));
```

Comment:
```suggestion
                    sentry::configure_scope(|scope| scope.set_transaction(Some("new_transaction")));
```
This will dispatch to the current hub